### DISCUSSION TITLE
Props `crop` and `onCropChange`

### DISCUSSION THREAD

**Discussion by mkosir** [ValentinH/react-easy-crop#487] [General]
Hey 👋
thanks for an awesome library 🚀
Probably I'm missing some use case, but just wondering, could [Cropper](https://github.com/ValentinH/react-easy-crop#basic-usage) component be uncontrolled, without having need to pass `crop` and `onCropChange` props?
Not sure what should be the use case of it, since I'm only keeping state of crop in my custom components that utilize Cropper.

**Comment by ValentinH** [comment]
I think that it could in theory be uncontrolled indeed. However, having it controlled brings more power to the end developer. 
I understand that we could support both controller and uncontrolled but I don't think this is important enough to add this logic to the library.
